armi/operators/tests/test_operators.py

```diff
@@ -103,37 +111,96 @@ def test_loadStateError(self):
         with self.assertRaises(RuntimeError):
             self.o.loadState(0, 1)
 
+    def test_setStateToDefault(self):
+
+        # reset the runType for testing
+        self.assertEqual(self.o.cs[CONF_RUN_TYPE], "Standard")
+        self.o.cs = self.o.cs.modified(newSettings={"runType": "fake"})
+        self.assertEqual(self.o.cs[CONF_RUN_TYPE], "fake")
+
+        # validate the method works
+        cs = self.o.setStateToDefault(self.o.cs)
+        self.assertEqual(cs[CONF_RUN_TYPE], "Standard")
+
+    def test_snapshotRequest(self):
+        with TemporaryDirectoryChanger():
+            self.o.snapshotRequest(0, 1)
+
+
+class TestTightCoupling(unittest.TestCase):
+    def setUp(self):
+        self.cs = settings.getMasterCs()
```

Comment:
did you try this with `settings.Settings()`?

Reply:
Yeah... See https://github.com/terrapower/armi/pull/1172#discussion_r1103260925

I decided that maybe `getMasterCs` would be the lesser of the two evils between it and `setMasterCs`.

The problem is that if I don't call `setMasterCs` when getting the case settings, this method rears its head and causes the test to fail. 
https://github.com/terrapower/armi/blob/867ebecaa4f508239da4c3b4137c1b89beba8fbd/armi/operators/operator.py#L548-L563

Reply:
@john-science are you ok with the `getMasterCs` route? We'd just have to add it to the to-do list for #930.

Reply:
Oh, darn! I did not read that clearly

I vote we get this in and you add it to the todo list. `setMasterCs` is used in every single unit test that calls `loadTestReactor`....so I'd imagine getting rid of it will take some work. Since there isn't a clear path to that yet, it shouldn't stop your test from being included in the code.

Reply:
I added it to https://github.com/terrapower/armi/issues/930